### PR TITLE
Update distributed docs

### PR DESCRIPTION
**Pull request by nitisht**
## Description
We need to clarify that distributed Minio doesn't strictly need a
fresh drive, instead Minio needs a fresh directory on the drive.

## Motivation and Context
Based on slack discussion with users

## How Has This Been Tested?
NA

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added unit tests to cover my changes.
- [ ] I have added/updated functional tests in [mint](https://github.com/minio/mint). (If yes, add `mint` PR # here: )
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by harshavardhana — DISMISSED** [review]

File: docs/distributed/README.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ To start a distributed Minio instance, you just need to pass drive locations as
 *Note* 
 
 - All the nodes running distributed Minio need to have same access key and secret key for the nodes to connect. To achieve this, you need to export access key and secret key as environment variables on all the nodes before executing Minio server command.
-- Disks used for Minio distributed should be fresh with no pre-existing data. 
+- Minio distributed mode requires fresh directories. If required, the drives can be shared with other applications. You can do this by using a sub-directory exclusive to minio. For example, if you have mounted your volume under `/export`, pass `/export/data` to Minio server arguments.
```
Comment (deekoder) [inline docs/distributed/README.md]: , pass /export/data as arguments to Minio server.

**Review by deekoder — DISMISSED** [review]
Lgtm except for this one comment.

File: docs/distributed/README.md (hunk 1 above)
Comment (nitisht) [reply docs/distributed/README.md]: updated @deekoder

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-gateway-azure.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |

**Review by harshavardhana — APPROVED** [review]

**Review by deekoder — APPROVED** [review]

**Pull request merged**